faq/java.inc

```diff
@@ -128,3 +128,22 @@ C/Fortran code).
 We suggest users experiencing performance problems to contact the
 <a href=\"https://www.open-mpi.org/community/lists/ompi.php\">
 OMPI users mailing list</a>.";
+
+/////////////////////////////////////////////////////////////////////////
+
+$q[] = "What are the known limitations of the Java Bindings?";
+$anchor[] = "java_limitations";
+
+$a[] = "There are currently issues with Intel Truescale (PSM) and Omnipath
```

Comment:
@nrgraham23 @hppritcha @matcabral Can you revise this a bit?  It's a bit weird to say "currently" in an FAQ item (because FAQ items live for a long, long time).  Can you be a bit more specific about which versions of PSM / PSM2 had problems, and which versions fixed the problem?

Also, there's inconsistencies about the capitalization of PSM in this entry -- can you make it consistent?

(I know this PR has already been merged, but it would be nice if a subsequent PR/commit could clarify / fix the language a bit -- thanks!)